### DISCUSSION TITLE
UltraSound Voyager  to Plus Server

### DISCUSSION THREAD

**Discussion by msepty** [PlusToolkit/PlusLib#1082] [General]
Hello 
i have an UltraSound Voyager Portable , ( not from the liste of ultrasound supported by Plusserver )

how i can add it in the Plus Server to use with 3dslicer application?

**Comment by ungi** [comment]
Hi, if your ultrasound has a video output, then you could use a USB frame grabber that PLUS supports, like Epiphan. That would capture the whole screen of the ultrasound machine, and you could crop out the ultrasound part by setting up the PLUS config file. If your ultrasound machine does not have a video output, then you could ask the vendor if they have any kind of data streaming interface that we could add to PLUS. If there is no video output and no data streaming option, then I think you need a different ultrasound for PLUS.
